src/cogs/jaegeraccounts.py

```diff
@@ -189,14 +191,14 @@ async def insert_bookings(self, bot: commands.Bot, ctx: commands.Context, url: s
             last_date = today
             last_index = len(raw_dates) + 1
             create_new_column = True
-
+        
         # Add new date column if needed
         if create_new_column:
             await bot.loop.run_in_executor(None, self._write_sheet_data, url, 1, last_index, last_date.strftime("%m/%d/%Y"))
 
         # Prepare data to write
         hrs_now = now.strftime("%I:%M%p")
-        hrs_after_x = (now + datetime.timedelta(hours=book_duration)).strftime("%I:%M%p")
+        hrs_after_x = (now + datetime.timedelta(hours=book_duration)).strftime("%I:%M%p")        
```

Comment:
Trailing whitespace, probably from old code.
```suggestion
        hrs_after_x = (now + datetime.timedelta(hours=book_duration)).strftime("%I:%M%p")
```